src/renderer/screens/config/ConfigScreens/WorkspaceScreen.js

```diff
@@ -120,9 +122,43 @@ class WorkspaceScreen extends Component {
           <h4>TRUFFLE PROJECTS</h4>
           <div className="Row">
             <div className="RowItem">
+              {this.props.config.validationErrors["workspace.project"] && (
```

Comment:
This is pretty hard to read, might be worthwhile to assign this to a more semantic variable first, like:

```js
const validationErrors = this.props.config.validationErrors["workspace.project"]
```

And then use it like:

```js
{validationErrors && <div>...</div>}
```

This would also simplify line 129 to simply be: `validationErrors.message`